Core/Src/pdu.c

```diff
@@ -5,21 +5,62 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#define PUMP_CTRL      0
-#define RADFAN_CTRL    1
-#define BRKLIGHT_CTRL  2
-#define BATBOXFAN_CTRL 3
-#define RTDS_CTRL      7 // PORT 17 BANK 1 (so read with 1_REG)
+#define PUMP_CTRL	0
+#define MPU_FAULT	2
+#define BRKLIGHT_CTRL	3
+#define FANBATTBOX_CTRL 4
+#define RTDS_CTRL	7 // PORT 17 BANK 1 (so read with 1_REG)
+// #define RADFAN_CTRL    1
 // #define TSMS_CTRL	   0x04
 // #define SMBALERT	   0x05
 #define MUTEX_TIMEOUT osWaitForever /* ms */
 
-#define SHUTDOWN_ADDR PCA_I2C_ADDR_3
-#define CTRL_ADDR     PCA_I2C_ADDR_2
+#define SHUTDOWN_ADDR PCA_I2C_ADDR_1
```

Comment:
Just want to confirm u made these changes on purpose to the i2c is of pca?

Reply:
Note for later in case they change:

Right now in Altium: ctrl i2c is 1110100 (0x74), and shutdown i2c is 1110101 (0x75)
Then in pca9539.h: PCA_I2C_ADDR_0 is 0x74, and PCA_I2C_ADDR_1 is 0x75